pkg/commands/app.go

```diff
@@ -0,0 +1,88 @@
+package commands
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aquasecurity/trivy-aws/pkg/scanner"
+	"github.com/aquasecurity/trivy/pkg/flag"
+	"github.com/aquasecurity/trivy/pkg/log"
+	"github.com/aquasecurity/trivy/pkg/types"
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+)
+
+func NewCmd() *cobra.Command {
+	reportFlagGroup := flag.NewReportFlagGroup()
+	compliance := flag.ComplianceFlag
+	compliance.Values = []string{
+		types.ComplianceAWSCIS12,
+		types.ComplianceAWSCIS14,
+	}
+	reportFlagGroup.Compliance = &compliance // override usage as the accepted values differ for each subcommand.
+	reportFlagGroup.ExitOnEOL = nil          // disable '--exit-on-eol'
+	reportFlagGroup.ShowSuppressed = nil     // disable '--show-suppressed'
+
+	awsFlags := &flag.Flags{
+		GlobalFlagGroup:  flag.NewGlobalFlagGroup(),
+		AWSFlagGroup:     flag.NewAWSFlagGroup(),
+		CloudFlagGroup:   flag.NewCloudFlagGroup(),
+		MisconfFlagGroup: flag.NewMisconfFlagGroup(),
+		RegoFlagGroup:    flag.NewRegoFlagGroup(),
+		ReportFlagGroup:  reportFlagGroup,
+	}
+
+	services := scanner.AllSupportedServices()
+	sort.Strings(services)
+
+	cmd := &cobra.Command{
+		Use:     "aws [flags]",
```

Comment:
```suggestion
		Use:     "aws-scan [flags]",
```

The long description of the command should also be updated.

Reply:
I've updated it for now but I think we should finalize the plugin name first. I have `aws-scan` for now as I couldn't come up with anything better.

So `trivy aws-scan <flags>`.